src/static.ts

```diff
@@ -1,17 +1,38 @@
 import { NextFunction, Request, Response } from "express";
-import { getBufferFromMinio } from "./utils/minio";
+import {
+  backblazeClient,
+  getBufferFromBackblaze,
+  getBufferFromMinio,
+} from "./utils/minio";
 import { minioClient } from "./utils/minio";
+import { HeadObjectCommand } from "@aws-sdk/client-s3";
+import logger from "./logger";
 
 export const serveStatic = async (
   req: Request,
   res: Response,
   next: NextFunction
 ) => {
-  let stat;
+  let backblazeStat;
+  let minioStat;
+  logger.info(`static: ${req.params.bucket}/${req.params.filename}`);
+
   // FIXME: someone who's better at devops than me will have to figure out
   // how we can serve these directly from minio
   try {
-    stat = await minioClient.statObject(req.params.bucket, req.params.filename);
+    backblazeStat = await backblazeClient.send(
```

Comment:
While this works it feels pretty slow. I'm wondering if making the image public on backblaze and referencing it directly would be better for us and our bandwidth. 

I guess in that case we need to change the `generateFullStaticImageUrl` url to reference backblaze, in which case it might make sense to store on our image object where it is stored, which would be a boolean I guess?

Reply:
After some deep debugging it looks like docker was having issues resolving addresses outside of its own network, so when I changed the docker DNS to just be the google IP (and not include 10.0.0.2 as it was on my computer). This seemed to work much faster.